File: backend/semgrep_rules/tests/python/kisa_sw_29_insecure_cookie.py
def unsafe(response):
    # ruleid: kisa.sw29.python.insecure-cookie-flags
    response.set_cookie("session", "value", secure=False)
def safe(response):
    # ok: kisa.sw29.python.insecure-cookie-flags
    response.set_cookie("session", "value", secure=True, httponly=True, samesite="Lax")

File: backend/semgrep_rules/tests/javascript/kisa_sw_29_insecure_cookie.js
function unsafe(res) {
  // ruleid: kisa.sw29.javascript.insecure-cookie-flags
  res.cookie("session", "value", { secure: false });
}
function safe(res) {
  // ok: kisa.sw29.javascript.insecure-cookie-flags
  res.cookie("session", "value", { secure: true, httpOnly: true, sameSite: "lax" });
}
